dts/bindings/video/st,stm32-dcmi.yaml

```diff
@@ -0,0 +1,72 @@
+#
```

Comment:
I think the binding should be as close as possible to what's done in Linux kernel https://wiki.st.com/stm32mpu/wiki/DCMI_device_tree_configuration
https://github.com/STMicroelectronics/linux/blob/v6.1-stm32mp/Documentation/devicetree/bindings/media/st,stm32-dcmi.yaml

Reply:
Dear, @ajarmouni-st. 
I check how the binding is implemented on Linux. If I understudy correctly, the Linux defined some port's properties, such as `hsync-active` and `vsync-active`, for example, into `video-interfaces.yaml` file, instead of directly into `st,stm32-dcmi.yaml` as I did. However, I didn't find that file on Zephyr. Should I implement it?

https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/media/video-interfaces.yaml

Reply:
> Should I implement it?

No, I don't think it's necessary.

Reply:
Thank you for your help!